gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/InitialImageOperation.java

```diff
@@ -232,6 +232,16 @@ private GIIStatus reportGIIStatus() {
     }
   }
 
+  private boolean hasLockedBucket = false;
+
+  private void unlockTargetBucketLock(InternalDistributedMember recipient) {
+    if (region instanceof BucketRegion && hasLockedBucket) {
+      this.region.getDistributionManager().getLoggerI18n().info(LocalizedStrings.DEBUG, "Processing SnapshotBucketLockReleaseProcessor to "+ recipient);
+      SnapshotBucketLockReleaseMessage.send(recipient, this.region.getDistributionManager(), region.getFullPath());
```

Comment:
An exception, in this case, means either the destination member is not reachable or, this member is going down. All other will be a code bug. In such cases we will get a member departed callback to release the lock. 

For GII, we can add a flag to catch the exception in the case of none interrupted cases. Will change that code.